src/main/java/Duke.java

```diff
@@ -1,10 +1,25 @@
+import java.util.Scanner;
```

Comment:
In general, I quite like that the code seems very clean as most of the processing/parsing functions are abstracted on a high level.